CMakeLists.txt

```diff
@@ -29,8 +29,16 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 set(CMAKE_PREFIX_PATH "$ENV{OPENSCAD_LIBRARIES}")
 
 find_package(Eigen3 REQUIRED QUIET)
-message(STATUS "Eigen3: ${EIGEN3_VERSION_STRING}")
-include_directories(${EIGEN3_INCLUDE_DIRS})
+if(DEFINED EIGEN3_VERSION_STRING)
+  message(STATUS "Eigen3: ${EIGEN3_VERSION_STRING}")
+else()
+  message(STATUS "Eigen3: ${EIGEN3_VERSION}")
+endif()
+if(DEFINED EIGEN3_INCLUDE_DIRS)
```

Comment:
I think this check is superfluous - both FindEigen3 and Eigen3Config defines `EIGEN3_INCLUDE_DIR`